index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <h1 id="title">survey-form</h1> 
+  <p id="description">it is a survey-form that made by using html and css</p>
+</head>
+<link rel="stylesheet" href="styles.css">
+<body>
+  <div class="container">
+    <form id="survey-form">
+      <div class="labels">
+        <label id="name-label">name</label><input id="name" type="text" placeholder="Enter your name"  required><br/>
+        <label id="email-label">email</label><input id="email" type="email" placeholder="Enter your email"  required><br/>
+        <label id="number-label">number</label><input id="number" min="20" max="66"  type="number" placeholder="Age"required></div><br/>
+        <select id="dropdown">
+          <option>web development </option>
+          <option>ux/ui</option>
+          <option>digital markiting </option>
+        </select>
+        <div class="form-group">
+          <p>Would you recommend freeCodeCamp to a friend?</p>
+          <label>Definitely</label>
+          <br/>
+          <input
+          name="user-recommend"
+          value="definitely"
+          type="radio"
+          class="input-radio"
+          checked>
+          <label>Maybe</label><br/>
+          <input
+          name="user-recommend"
+          value="maybe"
+          type="radio"
+          class="input-radio">
+          <label>Not sure</label><br/>
+          <input
+          name="user-recommend"
+          value="not-sure"
+          type="radio"
+          class="input-radio">
+          <label>sure</label>
+          <br/>
+        </div>
+        <br/>
+        <div class="checkbox">
+          <input
+          name="checkbox1"
+          value="multiple"
+          type="checkbox"
+          class="checkbox">
+          <label>Front-end Projects</label>
+          <br/> 
+          <input
+          name="checkbox2"
+          value="back-end"
+          type="checkbox"
+          class="checkbox">
+          <label>back-end</label>
+          <br/>
+          <input
+          name="checkbox3"
+          value="ux/ui"
+          type="checkbox"
+          class="checkbox">
+          <label>ux/ui</label>
+        </div>
+        <div class="coment">
+          <p>Any comments or suggestions?
+          </p>
+          <textarea
+          id="comments"
+          lass="textarea"
+          name="comment"
+          placeholder="Enter your comment here...">
+        </textarea>
+      </textarea>
```

Comment:
@latifa-wakili 

- Well done now your code passes all checks on freecodecamp, but still you have some issues in this part of your code the closing tag for your 2 divs is not excited here, and besides that you added an extra `</textarea>` closing tag.
- And also your first `</textarea>` is not aligned with its opening tag.
- Solve these mistakes and mention me back here so I can check your project using my username @ahzamir.